frontend/src/app/team/team-list/team-list.component.spec.ts

```diff
@@ -44,6 +44,12 @@ describe('TeamListComponent', () => {
     ).toEqual('Teams');
   });
 
+  test('should have create team button with right text', () => {
```

Comment:
Beim TestBed das **RouterTestingModule** auch importieren. Sonst hat man folgende Logmeldung während dem Testen:
 _NG0303: Can't bind to 'routerLink' since it isn't a known property of 'img' (used in the 'TeamListComponent' component template)._